src/ScottPlot/Plottable/RadialGaugePlot.cs

```diff
@@ -0,0 +1,709 @@
+                    color = GaugeColors[i],
+                    lineWidth = 10,
+                    markerShape = MarkerShape.none
+                };
+                legendItems.Add(item);
+            }
+
+            return legendItems.ToArray();
+        }
+
+        /// <summary>
+        /// Needed as part of IPlottable in ScottPlot.ScottForm
+        /// </summary>
+        /// <returns></returns>
+        public AxisLimits GetAxisLimits() =>
+            (GaugeLabels != null) ? new AxisLimits(-3.5, 3.5, -3.5, 3.5) : new AxisLimits(-2.5, 2.5, -2.5, 2.5);
+
+        /// <summary>
+        /// Reduces an angle into the range [0°-360°]
+        /// </summary>
+        /// <param name="angle">Angle value</param>
+        /// <returns>Return the angle whithin [0°-360°]</returns>
+        private double ReduceAngle(double angle)
+        {
+            double reduced = angle;
+
+            if (angle > 360.0)
+                reduced -= 360 * (int)(angle / 360);
+            else if (angle < -360.0)
+                reduced += 360 * (int)(angle / 360);
+
+            return reduced;
+        }
+
+        /// <summary>
+        /// This is where the drawing of the plot is done
+        /// </summary>
+        /// <param name="dims">Plot dimensions</param>
+        /// <param name="bmp">Bitmap where the drawing is done</param>
+        /// <param name="lowQuality">Image quality</param>
+        public virtual void Render(PlotDimensions dims, Bitmap bmp, bool lowQuality = false)
+        {
+            int numGroups = DataRaw.Length;
+            double minScale = new double[] { dims.GetPixelX(1), dims.GetPixelY(1) }.Min();  // Not sure why, but GetPixelX(1) returns a reasonable dimension to draw the plot
+            PointF origin = new PointF(dims.GetPixelX(0), dims.GetPixelY(0));
+
+            using Graphics gfx = GDI.Graphics(bmp, dims, lowQuality);
+            using Pen pen = GDI.Pen(WebColor);
+            using Pen penCircle = GDI.Pen(WebColor);
+            using Brush labelBrush = GDI.Brush(GaugeLabelsColor);
+
+            float lineWidth = (LineWidth < 0) ? (float)(minScale / ((numGroups) * (GaugeSpacePercentage + 100) / 100)) : LineWidth;
+            float radiusSpace = lineWidth * (GaugeSpacePercentage + 100) / 100;
+            float gaugeRadius = numGroups * radiusSpace;  // By default, the outer-most radius is computed
+            float maxBackAngle = (GaugeDirection == RadialGaugeDirection.AntiClockwise ? -1 : 1) * (NormBackGauge ? (float)AngleRange : 360);
+
+            pen.Width = (float)lineWidth;
+            pen.StartCap = StartCap;
+            pen.EndCap = EndCap;
+            penCircle.Width = (float)lineWidth;
+            penCircle.StartCap = System.Drawing.Drawing2D.LineCap.Round;
+            penCircle.EndCap = System.Drawing.Drawing2D.LineCap.Round;
+
+            using System.Drawing.Font fontGauge = new(Font.Name, lineWidth * GaugeLabelsFontPct / 100, FontStyle.Bold);
```

Comment:
I think you can just use the font object that was given rather than creating a new one. I'm pretty sure you can still modify the font size and weight.

Reply:
Sure, both properties can be modified in the original font object.
However that font is from `ScottPlot.Drawing`, and the `fontGauge` is from `System.Drawing` since it's used extensively in the gauge labels text drawing (used by the Graphics object `gfx` inside `DrawTextOnCircle`, `MeasureCharactersInWord`, and `MeasureCharacters`).
I'm also unsure whether modifying the original font object, just for the gauge labels, would also affect other plot text like title or legend...

Reply:
> I think you can just use the font object that was given rather than creating a new one

It's a little deceptive since the `using` hides the full namespace, but @arthurits did it right because they needed to create a `System.Drawing.Font` from a `ScottPlot.Drawing.Font` 👍

Reply:
... although I will refactor it since there's a helper method for this

```cs
Font.Size = lineWidth * (float)FontSizeFraction;
using System.Drawing.Font fontGauge = GDI.Font(Font);
```

It's a can of worms so I won't discuss it here, but there's also a difference between fonts reported in Pt (System.Drawing default behavior) and Pixel size (ScottPlot default behavior). Making the code change above changes font size slightly breaking the hash tests I have in place, so I'll make this change last.